actix-http/src/lib.rs

```diff
@@ -6,7 +6,6 @@
 //! | `openssl`        | TLS support via [OpenSSL].                            |
 //! | `rustls`         | TLS support via [rustls].                             |
 //! | `compress`       | Payload compression support. (Deflate, Gzip & Brotli) |
-//! | `secure-cookies` | Adds for secure cookies. Enables `cookies` feature.   |
```

Comment:
Doesn't look like it belongs here?

Reply:
was missed from previous PR

Reply:
I suspected so but I'm not a big fan of throwing random stuff into an otherwise clean PR. 😅